common/config/environment.go

```diff
@@ -115,7 +115,7 @@ func (e *CommonEnvironment) ExtraResourcesTags() map[string]string {
 
 func EnvVariableResourceTags() map[string]string {
 	tags := map[string]string{}
-	lookupVars := []string{"TEAM", "PIPELINE_ID"}
+	lookupVars := []string{"TEAM", "PIPELINE_ID", "CI_PIPELINE_ID"}
```

Comment:
@usamasaqib we can deprecate `PIPELINE_ID` in ebpf tests, `CI_PIPELINE_ID` is defined out of the box by gitlab with the same value